frontend/src/main/java/puf/frisbee/frontend/view/StartView.java

```diff
@@ -48,45 +33,26 @@ public void init(StartViewModel startViewModel, ViewHandler viewHandler) {
 		this.startViewModel = startViewModel;
 		this.viewHandler = viewHandler;
 
+		this.topPanelController.init(startViewModel, viewHandler);
 		this.bottomPanelController.init(viewHandler);
 
-		this.labelGreeting.textProperty().bind(this.startViewModel.getLabelGreetingProperty());
-		this.buttonLoginRegister.visibleProperty().bind(this.startViewModel.getShowLoginRegisterButtonProperty());
-		this.buttonSettings.visibleProperty().bind(this.startViewModel.getShowSettingsButtonProperty());
-		this.buttonStart.visibleProperty().bind(this.startViewModel.getShowStartButtonProperty());
-
 		this.highscoreTable.setItems(startViewModel.getHighscoreTableItems());
 		this.highscoreScore.setSortType(TableColumn.SortType.DESCENDING);
 		this.highscoreTable.getSortOrder().add(highscoreScore);
 		this.highscoreTable.sort();
-	}
-
-	@FXML
-//	private void handleLoginRegisterButtonClicked(ActionEvent event) {
-//		this.viewHandler.openGameView();
-//	}
-	private void handleLoginRegisterButtonClicked(ActionEvent event) {
-		this.viewHandler.openRegistrationLoginView();
-	}
 
-	@FXML
-	private void handleLogoutButtonClicked(ActionEvent event) {
-		this.startViewModel.logout();
+		this.buttonStart.visibleProperty().bind(this.startViewModel.getShowStartButtonProperty());
 	}
 
 	@FXML
-	private void handleProfileButtonClicked(ActionEvent event) {
-		this.viewHandler.openProfileView();
+	private void handleRefreshButtonClicked(ActionEvent event) {
+		this.startViewModel.refreshData();
+		this.highscoreTable.sort();
 	}
 
 	@FXML
 	private void handleStartButtonClicked(ActionEvent event) {
+		this.startViewModel.resetTeamData();
```

Comment:
Wollen wir jedes Mal beim Start alles wieder von vorne beginnen? Ich dachte, wir laden erstmal automatisch das letzte Game? 🤔 oder missverstehe ich da was?

Reply:
In der StartViewModel wird geprüft, ob das Spiel zuletzt beendet wurde oder nicht. Wenn ja, wird neu gestartet ansonsten der bisherige Stand aufgenommen.

Reply:
Hm, Du hast Recht, … das stimmt gerade nicht ganz. Muss ich mir noch mal ansehen.

Reply:
Irgendwie muss ich dann aber doch auch den aktuellen Countdown speichern, sonst kann man ewig – durch Rückkehr bzw. Umweg über die StartView – das gleiche Level immer wieder spielen.

Reply:
Das fände ich erstmal ok - wir laden immer das letzte gespeicherte (beendete) level mit der damaligen Punktzahl - wenn ein Team immer mitten im Level beendet und das Level wieder von vorne beginnt, ist das doch ok? Die Punkte werden ja immer auf den Stand des letzten erfolgreichen Levels resettet, bis das nächste Level geschafft ist.

Reply:
Ah OK, wenn wir die Punkte zurücksetzen, dann passt das. Das hatte ich nicht bedacht.

Reply:
Allerdings sollte man nicht zurückkehren und weiterspielen können, wenn das Spiel nach dem dritten Level tatsächlich beendet ist. Also muss ich vielleicht doch irgendwo einen Status festhalten. Oder?

Reply:
Wenn es wirklich beendet wird, dann sollten wir das speichern (der finale "Du hast es geschafft Dialog) und dann von vorne anfangen - aber das ist ja noch nicht Teil von diesem Ticket, oder?

Reply:
Hab das auch schon eingebaut, auch die anderen Optionen. Sollte jetzt so funktionieren, dass nur noch bei Pause nach einem geschafften Level der aktuelle Stand der Dinge ausgehend von der StartView wieder aufgenommen wird. Alles andere, wie z.B. Beenden während des Spiels oder Beenden nach dem letzten Level führt ermöglicht danach nur noch ein neues Spiel.

Reply:
Es fehlt jetzt noch die Möglichkeit zum Start eines neuen Spiels, nachdem das alte Spiel bis zum Ende erfolgreich gespielt wurde. Eventuell müsste man hierzu ein neues Team anlegen, um den aktuell noch gültigen Highscore nicht zu überschreiben. Der START PLAYING-Button in der StartView müsste sich dann in einen NEW TEAM-Button verwandeln.